src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -16,6 +16,9 @@ cdef class ReactionAlgorithm(object):
     reaction algorithm by setting the standard pressure, temperature, and the
     exclusion radius.
 
+    Note: When setting the velocities of the particles according the the
+    Maxwell distribution the mass of all particles is assumed to equal 1.
```

Comment:
Yes, I would note that this is not a problem as long as a thermostat is acting which thermalizes the particle velocities correctly before sampling an observable which is depending on the velocities (like e.g. the kinetic parts of the pressure).

Reply:
But it can still break you integration, if you have a particle with high mass which now has a much to high momentum.

I would let the user decide whether this is poses a problem or not.